sample_file.py

```
@@ -0,0 +1,47 @@
+USERNAME = "admin2"
+PASSWORD = "password123"
```

Ruff

Possible hardcoded password assigned to: "PASSWORD"

(S105)